modules/ROOT/pages/manage-message-flows.adoc

```diff
@@ -0,0 +1,206 @@
+= Manage Message Flows
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+After your message flow is running and you are using it with your partners, you should not have to change it much.
+There are certain cases where you might need to make changes, for example:
+
+* You are no longer trading with a particular partner and need to delete the message flow you created for them.
+
+* You completed the testing for a deployed message flow in a pre-production environment, and now want to reclaim those previously allocated Mule runtime resources.
+
+After a message flow is deployed, you can test it, undeploy it, and delete it.
+
+== Test Message Flows
+
+You can test your message flow after you deploy it to ensure that you are receiving messages and any acknowledgments from your partners.
+You should ask your partner to confirm receipt of the transmission once you deploy it to your production environment.
+You can also test the flow by deploying it in a sandbox environment, then undeploying it, and then deploying it to a production environment.
+
+Test your flow in your production environment:
+
+. Drop an EDI file on the receiving endpoint.
```

Comment:
We should say "initiate a transmission" or similar since "drop an EDI file" refers to SFTP only.

Reply:
Great point, thanks.